doc/nrf/gs_installing.rst

```diff
@@ -177,26 +190,32 @@ Install west
 
 To manage the combination of repositories and versions, the |NCS| uses :ref:`Zephyr's west <zephyr:west>`.
 
-To install west, enter the following command:
+To install west, complete the following steps:
 
 .. tabs::
 
    .. group-tab:: Windows
 
+      Open a command window with administrator rights in the :file:`ncs` folder and enter the following command:
+
       .. parsed-literal::
          :class: highlight
 
          pip3 install west
 
    .. group-tab:: Linux
 
+      Open a terminal window with administrator rights in the :file:`ncs` folder and enter the following command:
+
       .. parsed-literal::
          :class: highlight
 
          pip3 install --user west
 
    .. group-tab:: macOS
 
+      Open a terminal window with administrator rights in the :file:`ncs` folder and enter the following command:
```

Comment:
Probably the same comment as for Linux, but I don't know macOS.

Reply:
A Matter dev who installed NCS on macOS doesn't recall any special admin-related issues.